test/runtests.jl

```diff
@@ -87,3 +88,5 @@ function f5()
 end
 
 @test f4() == 42
+
+use_color(true)
```

Comment:
restoring it to true isn't always accurate, better to save the original value and restore that